src/main/java/hudson/plugins/sshslaves/RemoteLauncher.java

```diff
@@ -140,4 +141,9 @@ private String makeCommandLine(List<String> cmd, FilePath _workDir) {
     public void kill(Map<String, String> modelEnvVars) throws IOException, InterruptedException {
         // no way to do this
     }
+
+  @Override
+  public boolean isUnix() {
+    return File.pathSeparatorChar == ':';
```

Comment:
Is this better than checking `!Functions.isWindows()`?

Reply:
I did not know about that function, it is better to use it.